crates/sage-cli/src/main.rs

```diff
@@ -329,12 +330,21 @@ impl Runner {
                 &outputs.quant,
                 &filenames,
                 &self.database,
+                &self.parameters.annotate_matches,
             )?;
 
             let path = self.make_path("results.sage.parquet");
             path.write_bytes_sync(bytes)?;
             self.parameters.output_paths.push(path.to_string());
 
+            if self.parameters.annotate_matches {
+                let bytes =
+                    sage_cloudpath::parquet::serialize_matched_fragments(&outputs.features)?;
+                let path = self.make_path("results.matched.fragments.sage.parquet");
```

Comment:
I'm going to change the name to "matched_fragments.sage.parquet" for consistency with the tsv versions